orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -80,16 +80,37 @@ onnxruntime::MLDataType ort_scalar_type_from_aten(
 OrtValue create_ort_value(
   onnxruntime::ORTInvoker& invoker,
   const at::Scalar& scalar) {
-  // TODO: support more types
+  return create_ort_value(invoker, scalar, at::kFloat);
+}
+
+OrtValue create_ort_value(
+  onnxruntime::ORTInvoker& invoker,
+  const at::Scalar& scalar,
+  at::ScalarType type) {
   float val = scalar.toFloat();
+  at::BFloat16 valBFloat16 = scalar.toBFloat16();
```

Comment:
should we move the cast to the BFloat16 branch in the switch?

Reply:
thanks - moved.